sympy/printing/tests/test_conventions.py

```diff
@@ -81,6 +81,10 @@ def test_requires_partial():
     g = sum(f)
     assert requires_partial(Derivative(g, t)) is False
 
+
+@XFAIL
+def test_requires_partial_unspecified_variables():
+    x, y = symbols('x y')
     # function of unspecified variables
     f = symbols('f', cls=Function)
```

Comment:
I think the original intention was `f = symbols('f', cls=Function)(x)`.

Reply:
There is a comment there saying "function of unspecified variables" from which I conclude it was meant like this. But I added tests for applied undefined function, since there weren't any of that kind.